shared/constants/types/chat2/meta.tsx

```diff
@@ -17,6 +17,7 @@ export type PinnedMessageInfo = {
 }
 
 export type ConversationMeta = {
+  botAliases: {[key: string]: string}
```

Comment:
why not a `Map`?

Reply:
It just pastes in what it gets from the service. Otherwise we would need to build that `Map`, didn't seem worth it.